web/src/AutoBoundariesMode.svelte

```diff
@@ -222,8 +222,7 @@
       <div style="border: dashed black 2px; padding: 8px">
         {#if selectedBoundary}
           <div style="display: flex; justify-content: space-between;">
-            <b>Your Neighbourhood Overall</b>
-            <!--REVIEW: good place for a name input field? -->
+            <b>Your neighbourhood overall</b>
```

Comment:
iOS has a weird convention where you TitleCase all headings. I'm trying to unlearn it for web, but often forget. Feel free to correct me if you notice!